examples/webgl_geometry_sdf.html

```diff
@@ -0,0 +1,829 @@
+		mat4(1.66,.53,.54,.55,.37,-3.52,1.62,1.72,-.07,-.29,.15,.23,-.21,1.43,-.13,.21)*f1_1+
+		mat4(-.02,-.52,.14,.25,-.59,1.12,-.75,-.14,-.58,-.34,-.62,.31,-.58,1.11,.92,-.04)*f1_2+
+		mat4(-.17,-1.10,.31,-.94,-.96,.35,-.06,-.12,.38,-1.48,1.04,.27,-.47,1.09,-1.76,1.23)*f1_3+
+		vec4(.78,-1.51,-1.70,-.30))/1.4+f1_0;
+	vec4 f2_1=sin(mat4(1.92,-1.36,.28,1.21,.56,.98,-.21,-.71,.89,-.41,.31,1.48,.47,.09,-.72,-.13)*f1_0+
+		mat4(-.40,-.59,.50,1.09,-.88,.45,.23,-.72,-.09,-.26,-.04,.54,.47,-.20,.02,-.45)*f1_1+
+		mat4(-.21,-.27,-.06,.52,2.09,1.19,.96,.36,-.26,.02,.38,-.69,.42,-.43,-.72,.01)*f1_2+
+		mat4(2.63,.06,-.27,1.52,.53,.47,.07,.11,-.55,-.86,.10,.00,.07,1.05,.50,-.32)*f1_3+
+		vec4(-2.37,5.12,-1.09,-2.25))/1.4+f1_1;
+	vec4 f2_2=sin(mat4(.81,-.14,1.10,.17,.12,-.24,.31,-.08,.81,-.35,-.47,.16,.18,-.27,-.46,.32)*f1_0+
+		mat4(.27,.19,-.66,.24,.74,-.93,-2.85,.28,.11,.15,.26,.86,.32,-.02,.07,.01)*f1_1+
+		mat4(.08,.15,.21,.86,.69,.03,.11,.35,.01,-.31,-.80,-1.88,-.62,.30,.73,1.16)*f1_2+
+		mat4(.76,-.16,1.54,1.22,.13,.27,1.36,.66,-.33,-.12,-.83,-.93,1.23,.06,.11,-1.64)*f1_3+
+		vec4(-2.71,-1.53,.42,-.07))/1.4+f1_2;
+	vec4 f2_3=sin(mat4(-.16,-1.51,-.23,-.90,-.11,-.62,-.74,.63,1.37,-.79,.58,-.96,-.41,-.16,.00,.56)*f1_0+
+		mat4(-.18,.35,.01,-1.15,.07,1.33,1.03,1.26,.18,.63,.29,-.81,.61,.39,-.66,.70)*f1_1+
+		mat4(.12,.60,.35,-.86,.21,-.86,.17,-.52,-.03,.47,-.53,.57,-1.34,.21,1.09,-.35)*f1_2+
+		mat4(.56,-2.60,.27,-.68,.56,-1.01,-.05,.31,-.34,.71,-.07,-.36,-.32,1.49,-1.82,1.04)*f1_3+
+		vec4(-.41,.70,-.56,-1.46))/1.4+f1_3;
+	float d = dot(f2_0,vec4(.02,-.01,-.02,-.06))+
+		dot(f2_1,vec4(-.02,-.04,-.07,-.03))+
+		dot(f2_2,vec4(-.06,-.16,.03,.02))+
+		dot(f2_3,vec4(.06,-.03,-.04,.03))+
+		0.040;
+	// limit to inside unit sphere as neural sdf is not really defined
+	return max(d,length(p)-1.);
+}
+
+	</textarea>
+	<!-- Import maps polyfill -->
+	<!-- Remove this when import maps will be widely supported -->
+	<script async src="https://unpkg.com/es-module-shims@1.8.0/dist/es-module-shims.js"></script>
+
+	<script type="importmap">
+		{
+			"imports": {
+				"three": "../build/three.module.js",
+				"three/addons/": "./jsm/"
+			}
+		}
+	</script>
+	<script type="module">
+		import Stats from './jsm/libs/stats.module.js';
+		import { OrbitControls } from './jsm/controls/OrbitControls.js';
+		import { 
+			OrthographicCamera, 
+			Scene, 
+			WebGLRenderer, 
+			DataTexture, 
+			Clock, 
+			Mesh, 
+			Vector3, 
+			MeshDepthMaterial, 
+			AmbientLight, 
+			MeshStandardMaterial, 
+			Vector2, 
+			BufferGeometry, 
+			BufferAttribute, 
+			MeshBasicMaterial, 
+			PlaneGeometry, 
+			Matrix4, 
+			ShaderMaterial 
+		} from 'three';
+		
+		import { GUI } from './jsm/libs/lil-gui.module.min.js';
+		import { SDFtoMesh } from './jsm/geometries/SDFGeometry.js'
+		
+		let renderer = new WebGLRenderer( { alpha : true, preserveDrawingBuffer : true, antialias : false } );
+
+		class AppMain extends HTMLElement{
```

Comment:
Please do not organize the example like that but implement it like all others. Meaning without classes but simple `init()`, `animate()` `onWindowResize()` functions.

Reply:
✅